php-frontend/src/main/java/org/sonar/php/tree/symbols/DeclarationVisitor.java

```diff
@@ -171,4 +187,9 @@ private LocationInFile location(Tree tree) {
     return new LocationInFileImpl(filePath, firstToken.line(), firstToken.column(), lastToken.endLine(), lastToken.endColumn());
   }
 
+  private static boolean isFuncGetArgsCall(FunctionCallTree fct) {
+    return fct.callee().is(Tree.Kind.NAMESPACE_NAME)
+      && ((NamespaceNameTree) fct.callee()).fullyQualifiedName().equals("func_get_args");
```

Comment:
There is also `func_get_arg()` that can be an indicator of a dynamic-length arguments list.